client/src/components/_common/CommonUtils.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Checks that the field is a non-empty string
+ * @param {Object} props -
+ * @param {String} propName - name of the property
+ * @param {String} componentName - name of the component
+ * @returns {String} Message if error, otherwise null
+ */
+
+export default function isNotEmptyString(props, propName, componentName) {
+  if (
+    !props[propName] ||
+    props[propName] !== 'string' ||
```

Comment:
Should this actually be ?
`(typeof props[propName] === "string") `

Reply:
I think it should be 
`(typeof props[propName] !== 'string')`

Reply:
Thanks for catching that typo. Yep, it should be !==

Reply:
Replaced the code with: typeof props[propName] !== 'string', without () because it wasn't passing the linting test.